packages/sqlite/src/lib/Read.ts

```diff
@@ -0,0 +1,93 @@
+import { ITimelineEvent } from "@equinox-js/core"
+import { Format, LibSqlReader } from "./LibSqlClient.js"
+import { trace } from "@opentelemetry/api"
+import { Tags } from "@equinox-js/core"
+
+type StreamEventsSlice = {
```

Comment:
there's maxbatches/batchSize naming - is slice meaningfully different? should it have been called that?
slice is kinda analogous to pages of events from across docs
maybe it's all fine but the historic term may be just that?

Reply:
oh good point, I think the naming comes from the F# version for SqlStreamStore and has just kind of followed along, haven't actually put time into thinking about the names as it's entirely clear to me what the type represents :sweat_smile:

Reply:
yeah, the batch term came from Eirik over ESDB
and is a central name in cosmos and ddb
I reckon batch is for the best to reduce term count by one